index.js

```diff
@@ -95,8 +103,13 @@ var gs = {
     // then just pipe them to a single unique stream and return it
     var aggregate = new Combine(streams);
     var uniqueStream = unique('path');
+    var returnStream = aggregate.pipe(uniqueStream);
+
+    aggregate.on('error', function (err) {
```

Comment:
It seemed weird to me as well, but the tests fail without it: the error is just thrown instead of being emitted on the stream.